src/eventbrite-provider.ts

```diff
@@ -9,22 +11,63 @@ function EventbriteProvider(this: any, options: any) {
 
   let API_KEY = ''
 
+  let eventbriteSDK: Sdk
 
-  // seneca.message('role:entity,cmd:load')
-
+  seneca
+    .message('role:entity,cmd:load,base:eventbrite,name:event', loadEvent)
+    .message('role:entity,cmd:save,base:eventbrite,name:event', saveEvent)
 
 
   seneca.prepare(async function(this: any) {
     let out = await this.post('sys:provider,get:key,provider:eventbrite,key:api')
     if (out.ok) {
       API_KEY = out.value
+      eventbriteSDK = Eventbrite({token: API_KEY})
     }
     else {
       this.fail('api-key-missing')
     }
   })
 
+  async function loadEvent(this: any, msg: any) {
+    const q: any = msg.q
+    const eventID: string = q.id
+
+    const event: any = await eventbriteSDK.request(`/events/${eventID}`)
+
+    if(event.id) {
+      event.eventbrite_id = event.id
+      event.id = eventID
+    }
+
+    return this.make$('eventbrite/event').data$(event)
+  }
+
+  async function saveEvent(this: any, msg: any) {
+    const ent: any = msg.ent
+    const eventID: string = ent.id
+
+    const body = JSON.stringify({
+      event: {
+        description: {
+          html: ent.description.html,
```

Comment:
In eventbrite is the description stored as HTML? Do they provide a custom WYSIGWYG editor on their site?

Reply:
Yes, they provide a WYSIWYG editor. Reading the docs more carefully, I saw that event.description will be deprecated so I changed it to event.summary, according to the documentation.
However, I read there is a feature called Structured Content with which you can make multi media event descriptions based on modules. The POST request becomes a little bit more complex, but, considering the description to be summary + structured, should we provide it ? I can make a new PR providing it. Thank you !!
https://www.eventbrite.com/platform/docs/event-description